src/notification.scss

```diff
@@ -29,315 +29,154 @@ $block: #{$fd-namespace}-notification;
   $fd-notification-indicator-border-color-success: var(--sapSuccessBorderColor) !default;
   $fd-notification-indicator-border-color-information: var(--sapInformationBorderColor) !default;
 
-  $fd-notification-title-color: var(--sapGroup_TitleTextColor) !default;
-  $fd-notification-header-border-color: var(--sapPageHeader_BorderColor) !default;
-  $fd-notification-content-text-color: var(--sapContent_LabelColor) !default;
-
   // mixins
-  @mixin fd-priority-indicator($type) {
-    margin: 1px 0 0 0;
-    padding: 1rem 0.5rem 0 1rem;
 
+  @mixin fd-priority-indicator($type) {
     @include fd-rtl() {
-      padding: 1rem 1rem 0 0.5rem;
+      padding-right: 0;
+      padding-left: 0.5rem;
     }
-
     @include fd-icon("message-#{$type}");
   }
 
-  // ------------------------ CONTAINER ------------------------
-  @include fd-reset();
-
-  max-width: $fd-notification-max-width;
   background-color: $fd-notification-background-color;
   box-shadow: $fd-notification-box-shadow;
   border-radius: $fd-notification-border-radius;
-  border-left: 0.375rem solid;
-  border-left-color: $fd-notification-indicator-border-color-default;
-  margin-bottom: 0.5rem;
+  margin-bottom: 20px;
+  width: 100%;
+  max-width: $fd-notification-max-width-m;
+  box-sizing: border-box;
+  padding: 1rem 0.5rem 1rem 1rem;
 
   @include fd-rtl() {
-    border-left: none;
-    border-right: 0.375rem solid;
-    border-right-color: $fd-notification-indicator-border-color-default;
+    padding: 1rem 1rem 1rem 0.5rem;
```

Comment:
usually when we have different values for margins and padding left and right we put them in variables. This way if in future the value changes, you only need to update the variable

Reply:
In RTL only padding-left and padding-right should be updated